src/contexts/Config/Config.tsx

```diff
@@ -8,17 +14,42 @@ const ConfigContext = createContext<UseRequestResponse<Config>>({
   error: undefined,
 });
 
+const LOCAL_CONFIG = "construct-hub-config";
+
 export const useConfig = () => useContext(ConfigContext);
 
 export const ConfigProvider: FunctionComponent = ({ children }) => {
-  const [requestConfig, configResponse] = useRequest(fetchConfig);
+  const [configData, setConfigData] = useState<Config | undefined>(() => {
```

Comment:
Updates to this file are to store the previous config in localStorage before replacing with the resolved one. This just allows the app to be interactive a bit faster since the config can be requested in background